src/components/brain/PageBrainCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { Card, Center, Heading, HStack } from "@chakra-ui/react"
+import Link from "components/common/Link"
+import Image from "next/image"
+import slugify from "slugify"
+import { BrainCardData } from "types"
+type Props = {
+  pageData: BrainCardData
+}
+
+const PageBrainCard = ({ pageData }: Props): JSX.Element => {
+  const slugifiedTitle = slugify(pageData.title, { lower: true })
+
+  return (
+    <Link
+      href={`/brain/${slugifiedTitle}`}
+      prefetch={false}
+      _hover={{ textDecor: "none" }}
+      borderRadius="2xl"
+    >
+      <Card
+        borderRadius="2xl"
+        px={{ base: 5, sm: 6 }}
+        py="4"
+        w="full"
+        h="full"
+        background="rgba(63,63,70, 0.45)"
```

Comment:
I get that the gray color which we're using in other components was too light here, but could we try to use existing colors from our color palette here? E.g. `whiteAlpha.50` looks similar.